src/tokens.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import { _ } from 'lodash'
+import type {
+  DbApi,
+  Token,
+  WithTokens,
+  TokenWithDbId,
+} from 'icarus-backend'; // eslint-disable-line
+import { arraySum } from './helpers'
+
+const formatToken = (token: TokenWithDbId | Token): Token => ({
+  policyId: token.policyId,
+  assetName: token.assetName,
+  quantity: token.quantity,
+})
+
+const aggregateTokensForOnePolicy = (policyGroup: any, policyId: string) => _(policyGroup)
```

Comment:
it would be nice to give a type annotation to policyGroup to make it clear what kind of structure it is which would make the subsequent operations easier to follow

Reply:
also the `ForOnePolicy` suffix is imho unnecessarily verbose and `ForPolicy` should do as well

Reply:
Right, I forgot that. Will do